### ISSUE TITLE
Wallet notifications Chia 1.6.1

### ISSUE THREAD

**Issue by djerfy** [martomi/chiadog#355]
**Describe the bug**

No wallet notifications after upgrade to Chia v1.6.1

Log format has been changed, example:

```text
06:33:18.310 wallet chia.wallet.wallet_node    : INFO     request coin: 1fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffCoinState { coin: Coin { parent_coin_info: 1fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff, puzzle_hash: 1fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff, amount: 1750000000000 }, spent_height: None, created_height: Some(0000000) }
```

**Environment:**
 - OS: Ubuntu 20.04.5 LTS
 - Python version: 3.8.10
 - PIP version: 20.0.2
 - Chia version: 1.6.1
 - Chiadog version: 0.7.3
 - Harvester: local

**Comment by djerfy** [comment]
Not bad with this regex, real tests in progress (already good with unittest)

<img width="835" alt="image" src="https://user-images.githubusercontent.com/2886596/200166035-00c8915e-ac11-44ab-bb5e-a1baa59f1cb1.png">

**Comment by djerfy** [comment]
Issue closed. Implemented in release [v0.7.5](https://github.com/martomi/chiadog/releases/tag/v0.7.5)
